### ISSUE TITLE
Myxir is dead

### ISSUE THREAD

**Issue by Try** [Try/OpenGothic#22]
After Raven's death:
![изображение](https://user-images.githubusercontent.com/122894/63656886-78f93600-c79a-11e9-9660-b838a0ab6450.png)

Assume correct behavior is to teleport mage away from game-world.
Offended script:
```c++
if(RavenIsDead == TRUE)
{
	B_RemoveNpc(KDW_14030_Addon_Myxir_ADW);
};

func void B_RemoveNpc(var int npcInstance)
{
	var C_Npc npc;
	npc = Hlp_GetNpc(npcInstance);
	if(Hlp_IsValidNpc(npc) && !Npc_IsDead(npc))
	{
		npc.flags = 0;
		AI_Teleport(npc,"TOT"); // what if no such point?
		B_StartOtherRoutine(npc,"TOT");
		Npc_ChangeAttribute(npc,ATR_HITPOINTS,-npc.attribute[ATR_HITPOINTS_MAX]);
		AI_Teleport(npc,"TOT"); // why?!
	};
};
```

**Comment by Try** [comment]
```c++
void Npc::tick(uint64_t dt) {
   ...
  if(!checkHealth(false,true)){
    mvAlgo.aiGoTo(nullptr);
    mvAlgo.tick(dt);
    setOther(lastHit);
    aiActions.clear();    // <--- looks bad...
    tickRoutine(); 
    return;
    }
```

**Comment by Try** [comment]
fixed in 1.0.401
